graph/src/data/subgraph/mod.rs

```diff
@@ -542,6 +542,13 @@ pub struct BaseSubgraphManifest<C, S, D, T> {
     pub templates: Vec<T>,
     #[serde(skip_serializing, default)]
     pub chain: PhantomData<C>,
+    pub indexer_hints: Option<IndexerHints>,
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IndexerHints {
+    pub history_blocks: Option<BlockNumber>,
```

Comment:
@leoyvens how does this look? If we have ongoing pruning do we need to the `prune` hint to do the manual pruning?

```
indexingHints:
   prune:
    historyBlocks: 100
  ```